.mocharc.js

```diff
@@ -1,23 +0,0 @@
-'use strict';
```

Comment:
please revert this file

Reply:
will do - any idea why that happened?  this wasn't a line I manually deleted

Reply:
reverted

Reply:
Oh, I think that when you autosave the file it formatted it with eslint. I had the same thing happen to me.